### PR TITLE
[Mobile][NUI] Add videoview sample

### PR DESCRIPTION
**Pull request by pkalota**

### CODE REVIEW

File: Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
+﻿using Tizen.NUI;
```
Comment (dongsug-song) [inline Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs:1]: I think copyright comments are required here for public open source code.
could you please copy some from other source code?

File: Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+
+namespace VideoViewSample
+{
+    class Program : NUIApplication
+    {
+        private readonly string[] btnNames = { "play", "pause", "stop", "forward", "backward" };
```
Comment (dongsug-song) [inline Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs:9]: How about change "btnNames" as "buttonNames" ?

https://readlearncode.com/code-and-stuff/clean-code-variables/
: "Use words that can be pronounced and avoid abbreviations"

File: Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        void Initialize()
+        {
+            Window.Instance.BackgroundColor = Color.White;
+
+            player = new VideoView()
+            {
+                HeightResizePolicy = ResizePolicyType.SizeRelativeToParent,
+                WidthResizePolicy = ResizePolicyType.SizeRelativeToParent,
+                SizeModeFactor = new Vector3(1.0f, 0.5f, 1.0f),
+                ResourceUrl = DirectoryInfo.Resource + "sample.3gp",
```
Comment (dongsug-song) [inline Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs:27]: does this "sample.3gp" video file have no copyright issue?
I don't know so I am just asking. could you confirm?
thank you.

Comment (pkalota) [reply Mobile/VideoViewSample/VideoViewSample/VideoViewSample.cs:27]: It is file copied from an already existing sample - [Mobile.PlayerSample](https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/PlayerSample/PlayerSample/PlayerSample.Tizen.Mobile/res), so I assume file has no copyright issues

**Comment by dongsug-song** [comment]
Hello,
Thank you for your works done.

please check the links below and you can see SVACE, Comments Ratio checking, spelling checking, and etc are required.

https://code.sec.samsung.net/confluence/display/TNET/C%23+Sample+release+process

please check the guides and upload the results (any format would be OK) here.
all the checker hurdles are resolved, then we can merge this PR.
thank you.

**Pull request merged**